packages/manager/src/queries/cloudpulse/queries.ts

```diff
@@ -80,6 +80,6 @@ export const queryFactory = createQueryKeys(key, {
 
   token: (serviceType: string | undefined, request: JWETokenPayLoad) => ({
     queryFn: () => getJWEToken(request, serviceType!),
-    queryKey: [serviceType],
+    queryKey: [serviceType, { resource_ids: request.resource_ids }],
```

Comment:
```suggestion
    queryKey: [serviceType, request.resource_ids],
```

can't we put it like this?

Reply:
we can but this seems better for usequery in terms of clarity and caching - https://tanstack.com/query/v4/docs/framework/react/guides/query-keys

Reply:
okay fine